ocs2_log_path_rel: "dataset/ocs2/tmp/"
option: 1

## OPTION 0: Read the log
# NOTE: Provide the name without ".json"!
ocs2_log_name: "20230815_220339"

## OPTION 1: Read the folder of logs

n_round_digit: 2
save_prefix: ""

joint_arm_limit_min: []
joint_arm_limit_max: []